ui/src/components/IndividualCard.vue

```diff
@@ -112,8 +117,12 @@ export default {
   },
   computed: {
     getNameInitials: function() {
-      var names = this.name.split(" ");
-      var initials = names[0].substring(0, 1).toUpperCase();
+      const name =
+        this.name ||
+        this.identities[0].identities[0].username ||
+        this.identities[0].identities[0].email;
+      const names = name.split(" ");
+      let initials = names[0].substring(0, 1).toUpperCase();
```

Comment:
As I explained in my comment before, this isn't needed. The code should take the field `name` from the profile, not from the identities.